src/runtime_src/core/tools/xbutil2/SubCmdProgram.cpp

```diff
@@ -17,6 +17,7 @@
 // ------ I N C L U D E   F I L E S -------------------------------------------
```

Comment:
Copyright date (line 2) needs to be updated.

Reply:
comment addressed.